File: src/main/java/no/nav/domain/pensjon/regler/grunnlag/Barnekull.java
package no.nav.domain.pensjon.regler.grunnlag;

import java.io.Serializable;

public class Barnekull implements Serializable {


    /**
     * Antall barn i kullet.
     */
    int antallBarn;
    /**
     * Angir om persondetaljen brukes som grunnlag på kravet.
     */
    private boolean bruk;

}

File: src/main/java/no/nav/pensjon/regler/domain/grunnlag/Barnekull.kt
package no.nav.pensjon.regler.domain.grunnlag

import java.io.Serializable

class Barnekull : Serializable {
    /**
     * Antall barn i kullet.
     */
    var antallBarn = 0

    /**
     * Angir om persondetaljen brukes som grunnlag på kravet.
     */
    var isBruk: Boolean = true
}